### PR TITLE
Alerting: Add info to enable alert forwarding when using an external …

### PR DESCRIPTION
**Pull request by JohnnyQQQQ**
This PR adds an info text when enabling an Alertmanager datasource to receive external alerts, so that the user knows they also need to activate it on the admin page.

Screenshot 2023-05-26 at 15 16 43

### CODE REVIEW

File: public/app/plugins/datasource/alertmanager/ConfigEditor.tsx [hunk 1]
```diff
@@ -73,6 +73,11 @@ export const ConfigEditor = (props: Props) => {
             />
           </InlineField>
         </div>
+        {options.jsonData.handleGrafanaManagedAlerts && (
+          <span>
+            Make sure to enable the alert forwarding on the <a href="/alerting/admin">admin page.</a>
```
Comment (JohnnyQQQQ) [inline public/app/plugins/datasource/alertmanager/ConfigEditor.tsx:78]: I'm not sure if this is the best way to link internally.

File: public/app/plugins/datasource/alertmanager/ConfigEditor.tsx [hunk 2]
```diff
@@ -73,6 +73,11 @@ export const ConfigEditor = (props: Props) => {
             />
           </InlineField>
         </div>
+        {options.jsonData.handleGrafanaManagedAlerts && (
+          <span>
```
Comment (JohnnyQQQQ) [inline public/app/plugins/datasource/alertmanager/ConfigEditor.tsx:77]: I think we want the text to be a bit lighter, but I am not sure which class to use.

File: public/app/plugins/datasource/alertmanager/ConfigEditor.tsx (hunk 1 above)
Comment (gillesdemey) [reply public/app/plugins/datasource/alertmanager/ConfigEditor.tsx:78]: Use the `<Link to='/alerting/admin'></Link>` component from `react-router-dom` :)

File: public/app/plugins/datasource/alertmanager/ConfigEditor.tsx (hunk 2 above)
Comment (gillesdemey) [reply public/app/plugins/datasource/alertmanager/ConfigEditor.tsx]: I've sent you some pointers :)

**Review by gillesdemey — APPROVED** [review]
🚀

**Pull request merged**